js/augurs-prophet-js/src/lib.rs

```diff
@@ -189,6 +189,20 @@ impl Prophet {
             .add_regressor(name, regressor.unwrap_or_default().into());
         Ok(())
     }
+
+    /// Create dates to use for predictions.
+    /// Returns an error if the model has not been fit.
+    #[wasm_bindgen(js_name = "makeFutureDataFrame")]
```

Comment:
```suggestion
    #[wasm_bindgen(js_name = "makeFutureDataframe")]
```

I don't _think_ we're using `dataFrame` anywhere else in the code so for consistency let's keep `dataframe` as if it were one word.